openhub/management/commands/import_outside_committers_data.py

```diff
@@ -0,0 +1,66 @@
+import requests
+import json
+import xmltodict
+import logging
+
+from openhub.models import OutsideCommitter, ContributionsToPortfolioProject
+from django.core.management.base import BaseCommand
+from community.git import get_owner
+from openhub.oh_token import OH_TOKEN
+
+logging.basicConfig(level=logging.DEBUG)
+
+
+org_name = get_owner()
```

Comment:
not helpful to put this at the module level.

instead, add caching in the `get_owner` function, so that it can be called many times without performance problems.